fvtest/compilertest/tests/OpCodesTest.cpp

```diff
@@ -1298,14 +1298,6 @@ OpCodesTest::invokeDisabledOpCodesTests()
 void
 OpCodesTest::UnsupportedOpCodesTests()
    {
-   //bdiv, brem
-   addUnsupportedOpCodeTest(_numberOfBinaryArgs, TR::bdiv, "bDiv", _argTypesBinaryByte, TR::Int8);
-   addUnsupportedOpCodeTest(_numberOfBinaryArgs, TR::brem, "bRem", _argTypesBinaryByte, TR::Int8);
-
-   //sdiv, srem
-   addUnsupportedOpCodeTest(_numberOfBinaryArgs, TR::sdiv, "sDiv", _argTypesBinaryShort, TR::Int16);
-   addUnsupportedOpCodeTest(_numberOfBinaryArgs, TR::srem, "sRem", _argTypesBinaryShort, TR::Int16);
-
    //bucmplt, bucmple, bucmpgt, bucmpge
```

Comment:
Does anybody know of a valid reason to keep around the unsupported opcode tests for unsigned byte compares? It looks like those are the last of the tests in this particular suite and I don't personally see any reason we should be asserting that they're not supported by any codegens. Perhaps we could eliminate them as well?

Reply:
I agree. There's really no reason to keep these tests around. Removing them fits with the goal of eventually getting rid of all the tests in this directory.